label_sync/main.go

```diff
@@ -319,6 +319,10 @@ func GetOrg(org string) (string, bool) {
 	return org, false
 }
 
+// securityForkNameRE is a regexp matching repos that are temporary security forks
+// https://help.github.com/en/github/managing-security-vulnerabilities/collaborating-in-a-temporary-private-fork-to-resolve-a-security-vulnerability
+var securityForkNameRE = regexp.MustCompile(`^[\w-]+-ghsa-[\w-]+$`)
```

Comment:
it's a little unfortunate that we have to ignore some magical pattern :/

we should probably document this, however I'd prefer to get label_sync _working_ and have someone follow up

Reply:
I couldn't see any extra flag in the v3 API that calls out that this is a private security fork.

Reply:
same. IMO this is a reasonable fix to an unreasonable problem to have. It's not great that any repo with this string will magically be ignored but :man_shrugging:

Reply:
feedback for github really